app/controllers/sites/site/edit.js

```diff
@@ -25,6 +25,42 @@ function mapValidationDetailsToErrors(validationDetails) {
   }, {});
 }
 
+/**
+ *
+ * @param { import('../../models/address').AddressModel } addressModel
+ * @param { import('../../components/au-address-search').Address } addressSearchAddress
+ */
+function copyAddressSearchAddressToAddressModel(
```

Comment:
Copy from new.js. Refactor the conversion functions and put them into a separate file which both the new and edit controllers can import.